src/components/Form.jsx

```diff
@@ -29,28 +29,32 @@ function Form() {
   }
 
   return (
-    <>
-      <h2>Add  New Book</h2>
-      <form onSubmit={handleSubmit}>
+    <form action="" className="" method="" onSubmit={handleSubmit}>
+      <p className="form-title">Add New Book</p>
+      <div className="input-group">
         <input
+          style={{ color: 'black' }}
           type="text"
-          name="title"
+          placeholder="Book title"
           value={title}
+          id="text"
+          name="text"
           onChange={(e) => setTitle(e.target.value)}
-          id="title"
-          placeholder="Book Title"
+          className="title-input"
         />
         <input
+          style={{ color: 'black' }}
           type="text"
-          name="author"
+          placeholder=" Author"
           value={author}
           onChange={(e) => setAuthor(e.target.value)}
           id="author"
-          placeholder="Book Author"
+          name="author"
+          className="author"
         />
-        <Button className="add-book" type="submit" value="Submit" />
-      </form>
-    </>
+        <Button type="submit" className="add-btn" value="Add book" />
```

Comment:
Good work implementing the `form`

- Kindly changes the font to match Zeplin's design

Your design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/c372289c-b154-423e-9661-7ffd1a27d18d)


Zeplin design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/c7fb2f69-8516-40c8-8e62-e96b5e440571)

- Kindly add the grey line that's between the form and the list of books

Zepling design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/4fe31b8f-9510-4811-a997-84c3d3f46db4)